### ISSUE TITLE
Bot offline

### ISSUE THREAD

**Issue by PeskyPotato** [apurvakoti/Reddit-GIFSpeed-Bot#1]
Are there any plans to revive the bot in the near future?

**Comment by apurvakoti** [comment]
@LameLemon Yes! Planning to make it fully functional in a few weeks. Just because /u/GIFSpeedBot has gone on and off without much word, it'll probably start afresh from a new username once done.

**Comment by alexcroox** [comment]
Any updates on this?
